server.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+const app = express();
```

Comment:
It's a good practice to handle potential errors. Consider adding error handling middleware to your Express app. [important]